Typo fix in governance docs (#483)
centrehq/verite: packages/docs/verite/overview/governance-overview.md

* Defines how liability is allocated across various participants and network utilities 
* Define who has recourse and whom do they have recourse against and the terms of that recourse
* Representations and warranties of participants that signers commit to uphold

> * Technical standards that must be adhered to sustain and protect the interoperability / integrity of the ecosystem 

Reviewer: Technical standards must adherify unto their sustenance and protectorate their interoptitude

> * Technical standards that must be followed to sustain and protect the interoperability / integrity of the ecosystem 

Reviewer: ```suggestion
* Technical standards which, in being adhered to, sustain and protect the integrity of the ecosystem 
```
